### PR TITLE
no tiny fans, bad

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Remove tiny fans from the game

## Why It's Good For The Game
Tiny fans are the bane of atmos and should go away along with gas miners (in another PR)

## Changelog
:cl:
del: remove tiny fans
add: survival capsules doors have "holobarrier" to stop air from leaking
/:cl:

### CODE REVIEW

**Comment by itseasytosee** [comment]
aren't tiny fans in a ton of ruins?

**Comment by Ghilker** [comment]
yeah i'm gonna remove them all, i pr it so that travis will tell me where they are

**Comment by LemonInTheDark** [comment]
While they are annoying, they serve a purpose in quite a few places. Anything you add to replace their functionality will have the same issues they have.

**Comment by TemporalOroboros** [comment]
What will you be replacing the mining shelter fans with? One of their primary uses is a place to eat without choking on plasma.

**Comment by Ghilker** [comment]
well not really, i changed the doors of the capsules to have holobarriers that block air, similar to the holofans

**Comment by TemporalOroboros** [comment]
Alright, thank you.

**Comment by Fikou** [comment]
they shouldnt be removed on centcom areas

**Comment by Ghilker** [comment]
they have doors, there is no need for them to stay there

**Comment by RaveRadbury** [comment]
The Aux Base requires tiny fans, you'll need to modify the Aux Base droid.

**Comment by Ghilker** [comment]
i've removed it from aux_base_camera.dm there are other places too?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -76201,17 +76196,6 @@
 	},
 /turf/open/floor/plasteel,
 /area/security/checkpoint/engineering)
-"tpc" = (
-/obj/structure/fans/tiny/invisible,
-/obj/docking_port/stationary{
```
Comment (Qustinnus) [inline _maps/map_files/MetaStation/MetaStation.dmm:76206]: bruh

File: code/modules/mining/aux_base_camera.dm [hunk 2]
```diff
@@ -220,32 +212,6 @@
 		return
 	B.RCD.toggle_window_glass()
 
-/datum/action/innate/aux_base/place_fan
```
Comment (Qustinnus) [inline code/modules/mining/aux_base_camera.dm:223]: are you going to replace this with anything?

File: code/modules/mining/equipment/survival_pod.dm [hunk 3]
```diff
@@ -112,6 +112,7 @@
 	icon = 'icons/obj/doors/airlocks/survival/survival.dmi'
 	overlays_file = 'icons/obj/doors/airlocks/survival/survival_overlays.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_pod
+	CanAtmosPass = ATMOS_PASS_NO
```
Comment (Qustinnus) [inline code/modules/mining/equipment/survival_pod.dm:115]: fuck invisible atmos barriers,

all my homies hate invisible atmos barriers.

Comment (Qustinnus) [reply code/modules/mining/equipment/survival_pod.dm:115]: like how is this a better alternative to tiny fnas?

**Comment by nullAdept** [comment]
Why are they the bane of atmos?

**Comment by wesoda25** [comment]
What problem are you solving by removing these from areas such as centcomm? The place isn't supposed to be balanced. Tiny fans are necessary for mapping and what not, you haven't even given any justification in this PR what makes them so terrible in niche areas.

**Comment by itseasytosee** [comment]
Make them uncreatable maybe?

File: code/modules/mining/aux_base_camera.dm (hunk 2 above)
Comment (Ghilker) [reply code/modules/mining/aux_base_camera.dm:223]: aren't there doors?

File: code/modules/mining/equipment/survival_pod.dm (hunk 3 above)
Comment (Ghilker) [reply code/modules/mining/equipment/survival_pod.dm:115]: i added a holobarrier to the door, it makes more sense than a tiny fan that blocks all air by "a gust of wind"

File: _maps/map_files/MetaStation/MetaStation.dmm (hunk 1 above)
Comment (Ghilker) [reply _maps/map_files/MetaStation/MetaStation.dmm:76206]: bruh

**Comment by Paxilmaniac** [comment]
There is only one place you can actually make tiny fans, and that's in the aux base with the drone, the closest thing you can make to one is the atmos holofan. Like wesoda said, what problem is this actually fixing, how does this actually stop atmos players from doing something, because the only place these appear on station roundstart is at the escape pod docks, and some maps don't even have them.

**Comment by MrFagetti** [comment]
I was joking when I said coding for modern SS13 was hitting backspace.

**Comment by VolraSlava** [comment]
tenor

**Comment by MrFagetti** [comment]
How are these the bane of atmos, specifically? 90% of tiny fans are located in places where round-length atmos isn't exactly an option or in newer iterations of maps to stop the annoying items-to-space crud. Holofans are only capable of being made late-game (usually) and the most use I see them have is making SM setups from hell or stopping the entire hallway get vented when random funny event #464 happens. What exactly have these fans been used for that would require their removal entirely? I really want to know, since I'm really tired of half the PRs I see just being removing X thing cause Y. The only exploit I could see is some dunce using holofans to make a SM in Maint or something, but that should be something left to admins doing their damn job.

**Comment by msgerbs** [comment]
I am still curious what the problem with these are. I know it's been closed, but I want to understand what the motivation was in the first place if this is actually a problem.

**Pull request closed without merging**